src/main/slice/omero/Repositories.ice

```diff
@@ -129,14 +130,15 @@ module omero {
             void makeDir(string path, bool parents) throws ServerError;
 
             /**
-             * Similar to {@link #list} but recursive and returns only
+             * Similar to {@code list} but recursive and returns only
              * primitive values for the file at each location. Guaranteed for
              * each path is only the values id and mimetype.
              *
              * After a call to unwrap, the returned {@link omero.RMap} for a
              * call to treeList("/user_1/dir0") might look something like:
              *
              * <pre>
+             * {@code
```

Comment:
These seem to lose the indent. It complains with just the `<pre>` I guess?

Reply:
yes

Reply:
I'm not getting the complaint if I reverse it (OpenJDK 11.0.2) but I'm also not getting indents so there's no regression from my point of view so happy to let this stand in the absence of being able to compare with `slice2html`.